src/build/app/settings.rs

```diff
@@ -904,6 +907,36 @@ pub enum AppSettings {
     /// [``]: ./struct..html
     SubcommandRequiredElseHelp,
 
+    /// Specifies that the help subcommand should print the long form help message (if possible), instead of the short
+    /// form.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// # use clap::{App, AppSettings};
+    /// let m = App::new("myprog")
+    ///     .setting(AppSettings::SubcommandHelpShowsLongForm)
+    ///     .subcommand(App::new("test")
+    ///         .after_help("short form help msg")
+    ///         .after_long_help("long form help msg"))
+    ///     .get_matches();
+    /// // `myprog help test` will display the auto-generated help, as well as
+    /// // "long form help msg"
+    /// ```
```

Comment:
1. Let's remove the `after_` and use an arg with both `about` and `long_about`.
2. Let's remove `let m =`.
3. Let's remove the comment describing the behaviour. It should be explained in the docs not as comment in code.

Reply:
Fixed in latest commit